CHANGELOG.md

```diff
@@ -3,14 +3,28 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## TBD
+## [0.0.8] TBD
+### Added
+* Support to pass Graphite configuration to Shunting-Yard.
+
+## [0.0.7] 2019-01-28
+### Added
+* log4j.xml to be packaged with Shunting-Yard Binary.
+
+## [0.0.6] 2019-01-22
 ### Changed
 * Upgraded `hive` to 2.3.4 (was 2.3.0).
+* ContainerCredentialsProvider to a more robust EC2ContainerCredentialsProvider.
```

Comment:
These changes were made by me as and when I found a better way of doing things to test in Docker / ECS. So, I have updated the CHANGELOG for all of them together.